starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,16 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+  <meta charset="UTF-8">
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet"  href="css/main.css">  
 </head>
+
 <body>
 Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+
```

Comment:
You have to use <div> tags!!!!!
It's all about boxes inside boxes inside boxes. We'll go in deep now. Maybe (and for sure) you'll need to use block elements like <div>s instead of inline elements like <h1><p><img> to style websites